src/app/components/forms/post/post.ts

```diff
@@ -201,6 +204,11 @@ export class FormPost extends BaseForm<FormPostModel>
 		return this.longEnabled;
 	}
 
+	get tagText() {
+		const longText = this.longEnabled ? this.formModel.content_markdown : '';
+		return `${this.formModel.lead} ${longText || ''}`.toLowerCase();
```

Comment:
Maybe the tag-suggester component should lowercase, so that hte logic doens't need to be here.

Reply:
yes